frontend/components/panels/WalletDepositFlowCheckPanel.vue

```diff
@@ -81,7 +81,7 @@ export default Vue.extend({
         },
         tokenAddress: {
             type: String,
-            required: true,
+            default: 'Unknown token address.',
         },
```

Comment:
This will not be shown as far as I understand, instead it will be displayed as invalid url to etherscan. Instead, you need to make it empty string by default and handle it's representation in the `template` – instead of passing this address to the `<format-address>` you would need to display proper text that fits to the rest of the paragraph